src/components/BounceRuleDetailed/index.jsx

```diff
@@ -1,61 +1,531 @@
 import React from "react";
+import Breadcrumb from "@sendgrid/ui-components/breadcrumb";
+import CenterModal from "@sendgrid/ui-components/center-modal";
+import Button from "@sendgrid/ui-components/button";
+import { TextInput } from "@sendgrid/ui-components/text-input";
+import { Link } from "react-router-dom";
+import {
+  HeaderCell,
+  Table,
+  TableBody,
+  TableCell,
+  TableHeader,
+  TableRow,
+} from "@sendgrid/ui-components/table/table";
+import { Row } from "../Row";
+import { Column } from "../Column";
+import Header from "../Header";
+import Pagination from "../Pagination";
+import "./index.scss";
+import mockChangelog from "../../mocks/index";
 
-const BounceRuleDetailed = ({ currentRule }) => (
+const DetailContainer = ({ currentRule, handleButtonClicked }) => (
```

Comment:
We can change the naming to match later with DetailsContainer since the files will be changed to BounceRuleDetails